content/en/network_monitoring/devices/netflow.md

```diff
@@ -43,6 +43,27 @@ network_devices:
 
 After saving your changes, [restart the Agent][4].
 
+## Aggregation
+
+The Datadog Agent will automatically aggregate the received NetFlow data in order to limit the number of records sent to platform while maintaining most of the information. By default there is a 5 minutes aggregation interval, flow recording sharing the same identifying information (source and destination address and port, protocol etc.) will be aggregated together. Addtionnally the Datadog Agent can detect ephemeral port and remove them, this is why you may see Flows with `port:*`.
+
+## Enrichment
+
+Your NetFlow data is processed by the Datadog backend and they are enriched with the available metadata we have about your devices and interfaces. Enrichment is based on the NetFlow exporter IP and the interface indexes. In case of possible private IP collisions, a `namespace` configured on the Agent side is also used.
+
+In case the NetFlow exporter IP is one of the device IPs but not the one configured on the SNMP integration, Datadog will try to find which device does this exporter IP belong to and will enrich your NetFlow data with is as long as the match is unique.
+
+## Visualization
+
+You can find the NetFlow page directly from the [Network Devices page][5].
+{{< img src="network_device_monitoring/netflow/netflow_page.png" alt="NetFlow Page" >}}
+
+But your data is also available in Dashboards, Notebooks and more for more precise queries and for correlating with other sources of data.
```

Comment:
I think it can help to mention that the data is accessible via "NDM Traffic" data source.